plugins/modules/site_workflow_manager.py

```diff
@@ -1407,140 +1594,183 @@ def get_diff_merged(self, config):
-                self.log("Bulk operation status retrieved: {}".format(bulk_operation))
-                if bulk_operation:
-                    response = self.creating_bulk_site()
-                    self.log("Response from creating_bulk_site: {}".format(response))
-                    if response and isinstance(response, dict) and "response" in response:
-                        task_id = response["response"].get("taskId")
-                        if task_id:
-                            self.log("Task Id for the 'site_creation' task is {}".format(task_id), "INFO")
-
-                            task_name = "create_sites"
-                            success_msg = "Site created successfully."
-                            self.get_task_status_from_tasks_by_id(task_id, task_name, success_msg)
-                            self.result['changed'] = True
-
-                            name_list = [site.get("name") for site in self.want if "name" in site]
-                            self.created_site_list.append(name_list)
-                            self.log("Site '{}' created successfully".format(name_list), "INFO")
-
-                            for site in self.want:
-                                if site.get("type") == "floor":
-                                    floor_name = site.get("name")
-                                    self.log("Floor '{}' has been created successfully.".format(floor_name), "INFO")
-
-                                    upload_path = site.get("upload_floor_image_path", None)
-                                    if upload_path:
-                                        self.log(
-                                            "Upload path found for floor '{}'. Starting upload floor map from '{}.'".format(floor_name, upload_path), "INFO")
-                                        map_details, map_status, success_message = self.upload_floor_image(config)
-                                        if map_details:
-                                            self.log("Floor map for '{}' uploaded successfully: {}".format(floor_name, success_message), "INFO")
+                create_site = copy.deepcopy(self.handle_config["create_site"])
+                if len(create_site) > 0:
+                    for each_config in create_site:
+                        payload_data = self.change_payload_data(each_config.get("want"))
+                        if payload_data:
+                            payload_data[self.keymap["parent_name_hierarchy"]] =\
+                                payload_data.get(self.keymap["parent_name"])
+                            del payload_data[self.keymap["parent_name"]]
+
+                        if payload_data.get("type") == "area":
+                            self.handle_config["area"].append(payload_data)
+                        elif payload_data.get("type") == "building":
+                            self.handle_config["building"].append(payload_data)
```

Comment:
self.log("Added to building: {}".format(payload_data), "DEBUG")

Reply:
Added